### PR TITLE
DP-31525: Language in parentenses incorrect for docs on curated list

### PR DESCRIPTION
**Pull request by arthurbaghdas**
**Description:**
Explain the technical implementation of the work done.

**Jira:** (Skip unless you are MA staff)
DP-****

**To Test:**
- [ ] Add steps to test this feature

---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)

### CODE REVIEW

**Comment by joeg8612** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>Add a changelog YAML file to this PR</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 8f1ff233ff9cb568e696c57dd2b1dc12c25d9dba
</p>

File: docroot/modules/custom/mass_fields/mass_fields.module [hunk 1]
```diff
@@ -694,7 +694,9 @@ function mass_fields_paragraph_view(array &$build, EntityInterface $entity, Enti
       $item['#options']['manual_description'] = $build['field_listitemdoc_desc_manual'];
       $item['#field_name'] = 'extra_list_manual_desc';
       // Vary the cache by the SHA of the description so that updating the description updates the display.
-      $item['#cache']['keys'][] = sha1($entity->field_listitemdoc_desc_manual->value);
+      if (!$entity->get('field_listitemdoc_desc_manual')->isEmpty()) {
```
Comment (weitzman) [inline docroot/modules/custom/mass_fields/mass_fields.module]: Lets please refactor $entity->field_listitemdoc_desc_manual into a bundle class and edit at least this file so it uses the new method.

Comment (arthurbaghdas) [reply docroot/modules/custom/mass_fields/mass_fields.module]: Done.

File: docroot/modules/custom/mass_content/src/Entity/Bundle/paragraph/ListItemDocumentsBundle.php [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+namespace Drupal\mass_content\Entity\Bundle\paragraph;
+
+final class ListItemDocumentsBundle extends ParagraphBundle {
+
+  public function getManualDescription(): string {
+    if (!$this->get('field_listitemdoc_desc_manual')->isEmpty()) {
+      return $this->field_listitemdoc_desc_manual->value;
+    }
+    return '';
+  }
```
Comment (weitzman) [inline docroot/modules/custom/mass_content/src/Entity/Bundle/paragraph/ListItemDocumentsBundle.php]: Our other bundle classes dont return scalars like a string. We return the ListItem and then expect the caller to use methods like isEmpty() or getValue(). Lets stick with that pattern.

Comment (arthurbaghdas) [reply docroot/modules/custom/mass_content/src/Entity/Bundle/paragraph/ListItemDocumentsBundle.php]: Changed

**Pull request merged**